lib/spack/docs/tutorial/examples/1.package.py

```diff
@@ -30,8 +30,7 @@ class Mpileaks(AutotoolsPackage):
     MPI_Datatypes."""
 
     homepage = "https://github.com/hpc/mpileaks"
-    url      = "https://github.com/hpc/mpileaks/releases/download/v1.0/mpileaks-1.0.tar.gz"
-
+    url      = "https://github.com/hpc/mpileaks/releases/download/v1.0/mpileaks-1.0.tar.gz"  # NOQA
```

Comment:
NOQA shouldn't be needed for URLs.

Reply:
For some reason flake8 is checking it and giving me errors which is why I added NOQA